### PR TITLE
doc: document integrations with other CNCF and non-CNCF projects

### PR DESCRIPTION
**Pull request by luckymrwang**
#### What type of PR is this?
/kind documentation

#### Which issue(s) this PR fixes:

[#2189](https://github.com/openyurtio/openyurt/issues/2189)

#### Does this PR introduce a user-facing change?

```release-note

```

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
lgtm

File: versioned_docs/version-v1.4/integrations/integrations.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
```
Comment (rambohe-ch) [inline versioned_docs/version-v1.4/integrations/integrations.md:1]: how about add Chinese version of this docs?

Comment (rambohe-ch) [inline versioned_docs/version-v1.4/integrations/integrations.md:1]: please add this doc into Next version beside version-v1.4.

Comment (luckymrwang) [reply versioned_docs/version-v1.4/integrations/integrations.md:1]: @rambohe-ch Chinese version has been commited.

Comment (luckymrwang) [reply versioned_docs/version-v1.4/integrations/integrations.md:1]: @rambohe-ch This doc has been added into Next version.

**Review by rambohe-ch — APPROVED** [review]

**Pull request merged**